woocommerce/woocommerce-ios
# SwiftUI Notice does not render properly on iPad

Closes: #5905


## Description

- Added bottom padding to notices modifer

## Testing instructions

- test with both `.splitViewInOrdersTab` feature flag ON and OFF
- open new order and/or new customer
- try to get an API error (or fake it)
- check that the notice view looks ok

A way to fake a notice on new order screen, in `EditableOrderViewModel` `init` method add this to end of the function (around line 493):

```
self.fixedNotice = Notice(title: "API Error",
                  subtitle: "Restricted Access",
                  message: "Your photos could not be downloaded, please ask for the correct permissions!",
                  feedbackType: .error,
                  notificationInfo: nil,
                  actionTitle: "Retry",
                  actionHandler: {
                      print("Retry")
                  })
```

## Screenshots


Full screen notice|Modal notice
-|-
![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-01-24 at 12 09 12](https://github.com/woocommerce/woocommerce-ios/assets/6242034/533e1ad2-361c-4db6-9253-399ef692bb65)|![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-01-24 at 12 07 58](https://github.com/woocommerce/woocommerce-ios/assets/6242034/41009bce-5cf5-4faa-a21b-e18fe388e408)

---
